views/popups.pug

```diff
@@ -83,6 +83,10 @@
                 small
                   span.fas.fa-info-circle(title="New editors only (Reset UI to clear yours)" aria-label="New editors only (Reset UI to clear yours)")
                 select.defaultLanguage
+              .checkbox
+                label
+                  input.allowStoreCodeDebug(type="checkbox")
+                  | Allow my request to be stored (for up to a week) for diagnostic purposes in the event of an error
```

Comment:
I agree putting "temporarily" here is probably better. One less place to edit and remember to keep in sync with the PP